components/nodemanager-service/pgdb/project_delete.go

```diff
@@ -0,0 +1,89 @@
+package pgdb
+
+import (
+	"context"
+
+	project_update_lib "github.com/chef/automate/lib/authz"
+	uuid "github.com/chef/automate/lib/uuid4"
+	log "github.com/sirupsen/logrus"
+)
+
+const deleteProjectNodeAssociation = `
+DELETE FROM nodes_projects np
+USING projects p
+WHERE p.id = np.project_id AND p.project_id = $1;
+`
+
+type ProjectDelete struct {
+	jobStatus      project_update_lib.JobStatus
+	jobStatusError error
+	running        bool
+	ID             string
+}
+
+func (db *DB) DeleteProjectTag(ctx context.Context,
+	projectTagToBeDelete string) ([]string, error) {
+	jobUUID := uuid.Must(uuid.NewV4()).String()
+	log.Debugf("Running node manager DeleteProjectTag %v ID: %s", projectTagToBeDelete, jobUUID)
+
+	db.ProjectDelete.start(jobUUID)
+
+	go db.deleteProject(projectTagToBeDelete)
```

Comment:
We do not really need to run this delete async, but we will to keep the same pattern as the other services.